mak-etal-2015.ipynb

```diff
@@ -0,0 +1,1831 @@
+   "cell_type": "markdown",
+   "id": "fd70ac01",
+   "metadata": {},
+   "source": [
+    "## Cohort Preparation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "214a1194",
+   "metadata": {},
+   "source": [
+    "The first step in reproducing the paper is to build a PPMI cohort that looks the closest to Table 1. For this, we will use PPMI Study Data files."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e2fc3b14",
+   "metadata": {},
+   "source": [
+    "## PPMI study data files download"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b85b9c3e",
+   "metadata": {},
+   "source": [
+    "The required PPMI study files to build the cohort are:\n",
+    "\n",
+    "|Files|Used to retrieve|\n",
+    "|:---|:---|\n",
+    "|Demographics | gender|\n",
+    "|Age at visit | age|\n",
+    "|Socio-Economics| years of education|\n",
+    "|Participant Status | PD subjects and healthy controls|\n",
+    "|Cognitive Categorization | PD-MCI and PD-NC subjects| \n",
+    "|PD Diagnosis History | disease duration| \n",
+    "|MDS UPDRS Part III | Hoehn and Yahr scores|\n",
+    "|LEDD Concomitant Medication Log | LEDD scores|\n",
+    "|Montreal Cognitive Assessment MoCA | MoCA and MMSE scores|\n",
+    "|Geriatric Depression Scale Short Version | GDS scores|\n",
+    "\n",
+    "\n",
+    "We will use the LivingPark utils library to download these files from the notebook. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "425943b3",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Download skipped: No missing files!\n"
+     ]
+    }
+   ],
+   "source": [
+    "required_files = [\n",
+    "    \"Demographics.csv\",\n",
+    "    \"Age_at_visit.csv\",\n",
+    "    \"MDS_UPDRS_Part_III.csv\",\n",
+    "    \"Socio-Economics.csv\",\n",
+    "    \"LEDD_Concomitant_Medication_Log.csv\",\n",
+    "    \"Montreal_Cognitive_Assessment__MoCA_.csv\",\n",
+    "    \"PD_Diagnosis_History.csv\",\n",
+    "    \"Cognitive_Categorization.csv\",\n",
+    "    \"Participant_Status.csv\",\n",
+    "    \"Geriatric_Depression_Scale__Short_Version_.csv\"\n",
+    "]\n",
+    "\n",
+    "utils.download_ppmi_metadata(required_files)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d1ab4c82",
+   "metadata": {},
+   "source": [
+    "## LivingPark utility notebook download "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e2c1dfa7",
+   "metadata": {},
+   "source": [
+    "We will also import notebooks from LivingPark utils to clean data which produce the following files:\n",
+    "\n",
+    "* <code>MRI_info.csv</code>: contains 3D T1-weighted images by visit\n",
+    "* <code>MDS_UPDRS_Part_III_clean.csv</code>: contains cleaned-up PDSTATE and PDTRTMNT"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "54551ace",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# run.mri_metadata()"
```

Comment:
does this need to be commented out?

Reply:
I was going to bring that up in the next meeting. I commented it out for now because on my side I am still getting some errors even though I updated the version of ppmi_downloader. It keeps hanging at the log in.